### DISCUSSION TITLE
How to execute multiple runs simultaneously

### DISCUSSION THREAD

**Discussion by bounlu** [nextflow-io/nextflow#3144] [Q&A] [answered]
Hi,

I want to run 2 separate pipelines from my local on AWS, i.e. 1 smrnaseq and 1 rnaseq job.

However, I get the below error when I start the second run while the first one is still running.

How can I execute both runs simultaneously?

As I am using `awsbatch`, they should be safe to be submitted together to the same queue as there are plenty of resources.

```
N E X T F L O W  ~  version 22.04.5
Pulling nf-core/rnaseq ...
 Already-up-to-date
Launching `https://github.com/nf-core/rnaseq` [big_fermat] DSL2 - revision: 89bf536ce4 [master]
Unable to acquire lock on session with ID 04c3ff51-3742-4f93-aaba-12e9e8e638c1

Common reasons of this error are:
 - You are trying to resume the execution of an already running pipeline
 - A previous execution was abruptly interrupted leaving the session open

You can check what process is holding the lock file by using the following command:
 - lsof /Users/omeran/Desktop/xxx/github/nf-core/.nextflow/cache/04c3ff51-3742-4f93-aaba-12e9e8e638c1/db/LOCK
```

**Comment by bentsherman** [comment, marked as answer]
Hi @bounlu , if you want to run multiple workflows at the same time then you need to run each one in a separate directory. The `.nextflow` directory that gets created can only be used by one workflow at a time. They can use the same work directory though.
